pkg/authorization/api/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package api is the internal version of the API.
+// +genconversion=true
```

Comment:
@smarterclayton @deads2k I found that you need to have these in all packages in order to genconversion to succeed. Is this OK change? The output of the script looks like:

https://paste.fedoraproject.org/370270/46408680/

Also there were some changes in conversions, I can't say if they are OK or not either.